test/test_basic.py

```diff
@@ -229,6 +236,8 @@ def test_coords(request, capfd):
 
 
 def test_to_vtk(tmpdir_factory, request):
+    pass
```

Comment:
I think these added lines can go.

Reply:
Oops! that was there because I wanted to skip this test while I was iterating on additions... gone now

Reply:
Been there. Thanks!!